public/app/features/dashboard/dashgrid/PanelHeader/PanelHeader.tsx

```diff
@@ -131,7 +131,7 @@ export class PanelHeader extends Component<Props, State> {
   render() {
     const { panel, scopedVars, error, isViewing, isEditing, data, alertState } = this.props;
     const { menuItems } = this.state;
-    const title = templateSrv.replace(panel.title, scopedVars, 'text');
+    const title = getTemplateSrv().replace(panel.title, scopedVars, 'text');
```

Comment:
This component uses only the replace function, should be ok to just import `getTemplateSrv` from grafana/runtime?